app/views/api/v2/settings/show.json.rabl

```diff
@@ -1,3 +1,3 @@
-object SettingPresenter.from_setting(@setting)
+object Foreman.settings.find(@setting.name)
```

Comment:
Why is this needed? Can `@setting` be `nil` and thus end up calling `nil.name`?

Reply:
no, It would be caught by controller.
We are still operating with the model in controller, so this is the easiest how to be handling model in controller, but presenter for views.